src/App.tsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
+import HomePage from "./components/home-page/HomePage";
+import NavBar from "./components/nav-bar/NavBar";
+import {BrowserRouter, Switch, Route} from "react-router-dom";
 
-function App() {
-  return (
-    <div className="App">
-
-    </div>
-  );
-}
+const App: React.FC = () => {
```

Comment:
why do you define the function as constant?

Reply:
It's my code style